### ISSUE TITLE
win_nssm issue with no appParameters specified

### ISSUE THREAD

**Issue by baudren** [ansible/ansible-modules-extras#1161]
#### Issue Type
- Bug Report
#### Ansible Version

1.9.4
#### Ansible Configuration

Contents of `.ansible.cfg` in the root of my project.

``` ini
[defaults]
hostfile = ./hosts
host_key_checking = False
remote_user = ansible

[ssh_connection]
ssh_args = -o PasswordAuthentication=no -o ControlMaster=auto
pipelining = True
```

Using the ansible extra `win_nssm` (copied the `.ps1` and `.py` file).
#### Environment
- Debian 7.7 Wheezy to run Ansible
- Managing a Windows 7 Enterprise, SP1 (connection via a kerberos ticket)
#### Summary

When defining a service with `win_nssm` based on a simple `.bat` script file, running **without** any `appParameter` option leads to the following error:

```
msg: Cannot bind argument to parameter 'appParameters' because it is an empty string.
```
#### Steps To Reproduce

Example of a task:

``` yaml
- name: create a simple service
  win_nssm: >
    name=toto
    application={{home}}toto.bat
    state=present
```

The file `toto.bat` can be anything. The problem is that the `appParameters` option, when not specified, is set to `$null`. The error

```
msg: Cannot bind argument to parameter 'appParameters' because it is an empty string.
```

Appears when running the task. I found [these lines](https://github.com/ansible/ansible-modules-extras/blob/devel/windows/win_nssm.ps1#L62-L69) where, when unspecified, the value of `appParameters` is set to `$null`. Combined with the fact that this `appParameters` seems later on to be mandatory, this might confuse `NSSM`.
#### Expected Results

I am expecting to be able to launch a service without any `appParameters`.
#### Actual Results

A service without `appParameters` is not successfully initialized through `win_nssm`. By changing the default value of [this line](https://github.com/ansible/ansible-modules-extras/blob/devel/windows/win_nssm.ps1#L68) to `" "` (note the space between the quotes), the problem disappears.

**Comment by robynbergeron** [comment]
[module: windows/win_nssm.py]

@baudren Thanks for submitting this bug report.

@georgefrank @h0nIg --  thanks in advance for taking a look at this bug report and for your ongoing work in maintaining this module.

**Comment by georgefrank** [comment]
Taking a look now. Thanks for the bug report!

**Comment by nitzmahone** [comment]
Should be fixed by #1327, closing
